build_tools/python_deploy/build_linux_packages.sh

```diff
@@ -305,6 +305,9 @@ function test_in_tree() {
       echo ":::: Run Linalg e2e integration tests"
       python -m e2e_testing.main --config=linalg -v
 
+      echo ":::: Run Onnx e2e integration tests"
+      python -m e2e_testing.main --config=onnx -v
```

Comment:
Should this also be added to https://github.com/llvm/torch-mlir/blob/main/build_tools/ci/test_posix.sh ? Why add in one place but not the other? (I was looking at logs here to see if these tests were running: https://github.com/llvm/torch-mlir/actions/runs/7878784431/job/21497737773?pr=2795#step:8:660)

Reply:
Added

Reply:
Thanks! https://github.com/llvm/torch-mlir/actions/runs/7909977079/job/21591862670?pr=2795#step:8:5047
```
  Summary:
      Passed: 447
      Expectedly Failed: 714
```